packages/near-api-js/src/account_multisig.ts

```diff
@@ -381,6 +396,57 @@ export class Account2FA extends AccountMultisig {
         });
     }
 
+    /**
+     * Returns `false` if multisig cannot be disabled by calling disable() directly
+     * i.e. account has too many LAKs to delete and re-create as FAKs for a single transaction
+     */
+    async canDisableMultisig() {
+        const keys = await this.get2faLimitedAccessKeys();
+
+        // there are 3 actions required to disable 2FA on top of the key conversion (2 actions per key)
+        // floor((100 - 3) / 2) = (96 / 2) = 48 = LAK_CONVERSION_LIMIT - 2
+        return keys.length <= (LAK_CONVERSION_LIMIT - 2);
+    }
+
+    /**
+     * Converts multisig LAKs (excluding the active key in the wallet, provided here as the
+     * `signingPublicKey` argument) back to FAKs when there are too many key conversions to be
+     * executed within the same transaction.
+     *
+     * Note that this undermines the security provided by 2FA by converting existing multisig LAKs into FAKs
+     * capable of bypassing the 2FA prompt when signing transactions outside the wallet (transactions signed
+     * in the wallet will continue to require 2FA approval). This method should only be used to convert LAKs
+     * as a way to make 2FA disabling possible.
+     *
+     * To be deprecated after 2FA migration; this addresses a specific issue with multisig deployed via wallet.
+     * @param signingPublicKey the public key used to sign transactions in the wallet
+     */
+    async batchConvertKeys(signingPublicKey: string) {
+        await this.validateMultisigState();
+
+        let batch = 0;
+        const keysToConvert = (await this.get2faLimitedAccessKeys())
+            .filter(({ public_key }) => public_key !== signingPublicKey);
+
+        while (batch < keysToConvert.length) {
+            const conversionActions = keysToConvert
+                .slice(batch, batch + LAK_CONVERSION_LIMIT)
+                .reduce((conversionActions: Action[], { public_key }) => {
+                    conversionActions.push(deleteKey(PublicKey.from(public_key)));
+                    conversionActions.push(addKey(PublicKey.from(public_key), fullAccessKey()));
+
+                    return conversionActions;
+                }, [] as Action[]);
+
+            await this.signAndSendTransaction({
```

Comment:
Correct, it will call the overridden method defined in `Account2FA`: https://github.com/near/near-api-js/blob/master/packages/near-api-js/src/account_multisig.ts#L240